build.gradle

```diff
@@ -224,6 +224,36 @@ subprojects {
             }
         }
     }
+
+
+    task copyDependencies(type: Copy) {
+        from configurations.runtime
+        into new File(buildDir, "dex")
+    }
+
+    task copyLib(type: Copy, dependsOn: copyDependencies) {
+        from new File(buildDir, "libs")
```

Comment:
I wonder if there is a config variable for the `"libs"` dir, but we already use this string elsewhere so it's already established practice.